r2/example.ini

```diff
@@ -361,7 +366,22 @@ CLOUDSEARCH_DOC_API =
 CLOUDSEARCH_SUBREDDIT_SEARCH_API =
 # endpoint for subreddit upload
 CLOUDSEARCH_SUBREDDIT_DOC_API =
-
+#
+# endpoint for link search 
+SOLR_SEARCH_HOST = 127.0.0.1
```

Comment:
I'm a wary of adding unused config that isn't enabled by install-reddit.sh. My preference would be to leave these values unset in example.ini and document them appropriately here or in the provider implementation.